frontend/tests/jest/__mocks__/products.ts

```diff
@@ -1,5 +1,5 @@
 import { CurrencyCode, FindProductQuery } from '@lib/shopify-storefront-sdk';
-import { MenuItemType } from '@models/menu';
+import { MenuItemType } from '@ifixit/ui';
```

Comment:
```suggestion
import type { MenuItemType } from '@ifixit/ui';
```

Reply:
Surprisingly,  `MenuItemType` is not a type. It is an enum. So, the compiler complains about importing this as `type`.